Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/TestSession.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium;
+using System;
+
+namespace $safeprojectname$
+{
+    // In order to run test, please download and install Win App Driver from https://github.com/Microsoft/WinAppDriver/releases.
+    // Then run it in background, default it listens to 127.0.0.1:4723
+    // For more information, see https://github.com/Microsoft/WinAppDriver
+    public class TestSession
+    {
+        // Win App Driver runs on localhost:4723 by default
+        protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+        // Provide the app id of your application
+        private const string ApplicationID = @"Your Application ID";
```

Comment:
Is it clearly enough to the user they need to enter their application ID? @shyamnamboodiripad